File: src/test/java/io/bloco/faker/components/UniversityTest.java
package io.bloco.faker.components;

import org.junit.Before;
import org.junit.Test;

import io.bloco.faker.Faker;

import static org.junit.Assert.assertNotNull;

public class UniversityTest {

    private Faker faker;

    @Before
    public void setUp() {
        faker = new Faker();
    }

    @Test
    public void name() {
        assertNotNull(faker.university.name());
    }

    @Test
    public void prefix() {
        assertNotNull(faker.university.prefix());
    }

    @Test
    public void suffix() {
        assertNotNull(faker.university.suffix());
    }
}

File: lib/src/test/kotlin/io/bloco/faker/components/UniversityTest.kt
package io.bloco.faker.components

import io.bloco.faker.Faker
import junit.framework.TestCase.assertNotNull
import org.junit.Before
import org.junit.Test

class UniversityTest {
    private val faker = Faker()

    @Test
    fun name() {
        assertNotNull(faker.university.name())
    }

    @Test
    fun prefix() {
        assertNotNull(faker.university.prefix())
    }

    @Test
    fun suffix() {
        assertNotNull(faker.university.suffix())
    }
}